app.py

```
@@ -1,9 +1,12 @@
 import streamlit as st
 from pathlib import Path
+from src.common.common import captcha_control  # 🔥 Import Captcha
```

Ruff (0.8.2)

`src.common.common.captcha_control` imported but unused

Remove unused import: `src.common.common.captcha_control`

(F401)